src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -455,6 +457,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                             End If
                             Continue For
 
+                        Case "refout"
+                            Dim unquoted = RemoveQuotesAndSlashes(value)
+                            If String.IsNullOrEmpty(unquoted) Then
```

Comment:
That's on purpose. "/out" could update the `outputDirectory`, but no other parameter can do that. So "/refout" should not either. "/refout" behaves like "/doc" or "/errorlog".